src/cases/store/reducers/search-filter.reducer.spec.ts

```diff
@@ -0,0 +1,26 @@
+import * as fromCases from './search-filter.reducer';
+import * as fromActions from '../actions/case-search.action';
+
+describe('SearchFilterReducer', () => {
+  describe('undefined action', () => {
+    it('should return the default state', () => {
+      const { initialStateSearch } = fromCases;
+      const action = {} as any;
+      const state = fromCases.reducer(undefined, action);
+
+      expect(state).toBe(initialStateSearch);
+    });
+  });
+
+  describe('APPLIED action', () => {
+    it('should apply patload to store', () => {
+      const { initialStateSearch } = fromCases;
+      const payload = {};
+      const action = new fromActions.Applied(payload);
+      const state = fromCases.reducer(initialStateSearch, action);
+      expect(state.loading).toEqual(false);
+      expect(state.loaded).toEqual(false);
+    });
```

Comment:
can you add test for serachFilterCiteria property and the loaded state should change to true